### PR TITLE
Added a kubernetes profile to init command (CRAFT-1318).

### PR DESCRIPTION
**Pull request by facundobatista**
Also fixed "simple" template (don't know why these changes didn't get into the previous PR).

### CODE REVIEW

File: charmcraft/templates/init-kubernetes/metadata.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# This file populates the Overview on Charmhub.
+# See https://juju.is/docs/some-url-to-be-determined/ for a checklist and guidance.
```
Comment (jnsgruk) [inline charmcraft/templates/init-kubernetes/metadata.yaml.j2:2]: We should remove this line
```suggestion
```

**Review by jnsgruk — APPROVED** [review]

File: charmcraft/templates/init-kubernetes/metadata.yaml.j2 (hunk 1 above)
Comment (facundobatista) [reply charmcraft/templates/init-kubernetes/metadata.yaml.j2:2]: done!

File: charmcraft/templates/init-kubernetes/.gitignore.j2 [hunk 2]
```diff
@@ -0,0 +1 @@
+../init-simple/.gitignore.j2
```
Comment (tigarmo) [inline charmcraft/templates/init-kubernetes/.gitignore.j2:1]: so does this syntax mean to use the contents of the other file? i.e., is it a shortcut to Jinja2's `include`?

**Review by tigarmo — APPROVED** [review]

File: charmcraft/templates/init-kubernetes/.gitignore.j2 (hunk 2 above)
Comment (facundobatista) [reply charmcraft/templates/init-kubernetes/.gitignore.j2:1]: simpler! Jinja2 just sees the file, and for the human checking it it's clearly the same file...

**Pull request merged**